frontend/app_flowy/lib/workspace/presentation/plugins/doc/document.dart

```diff
@@ -179,6 +179,10 @@ class DocumentShareButton extends StatelessWidget {
         switch (action) {
           case ShareAction.markdown:
             context.read<DocShareBloc>().add(const DocShareEvent.shareMarkdown());
+            const BubbleNotification(
+                    msgTitle: 'Exported Complete ^_^',
```

Comment:
It should support i18n. Check out this:
![image](https://user-images.githubusercontent.com/86001920/171071856-6b8d34d6-2243-4237-b0d4-4cd000c046b8.png)

After you config the en.json, run the gen language task. You can call:
LocaleKeys.your_key.tr()